webpack/assets/javascripts/foreman_settings.js

```diff
@@ -0,0 +1,14 @@
+import $ from 'jquery';
+
+$(document).ready(() => {
+  if (window.location.pathname === window.foreman_url('/settings')) {
+    $('.editable').editable({
```

Comment:
this will register the callback each time you go to a screen like that, is there a chance it will create multiple on click events or similar?